scala-parallel-collections/src/main/scala/zio/prelude/scalaparallelcollections/Syntax.scala

```diff
@@ -13,7 +13,7 @@ trait Syntax {
     )(r: ParMap[K, V]): PartialOrdering = {
       def commonValues(lesserMap: ParMap[K, V]): ParIterable[(V, V)] =
         // `toIterable` so that we don't incidentally create a map (and thus drop duplicate would-be keys)
-        lesserMap.toIterable.map { case (k, _) => (l(k), r(k)) }
```

Comment:
You just deleted this without addressing the comment above explaining why it was added.

Reply:
Oh yes..My bad. Reverting this change. Thanks